data/tiles.yml

```diff
@@ -41,3 +41,75 @@ test_analytics_guides:
         url: "/docs/test-analytics/your-own-collectors"
       - text: "CI environment variables"
         url: "/docs/test-analytics/ci-environments"
+  - ""
+integrations_row1:
+  - title: "Source Control"
+    desc: "Buildkite integrates with several popular code repository management systems. These integrations allow you to trigger builds based on version control events."
+    url: "/docs/integrations"
+    links:
+      - text: "GitHub"
```

Comment:
Might be nice to add a oneliner description to these, what do you think? (doesn't have to block this PR)

Reply:
Yeah sounds good, happy to raise a separate PR to add the oneliners. This PR is very bare, mainly trying to resolve the weird redirect behavior 🐛 💥 

What do you have in mind for the oneliner? Something like the expressing the simplicity of integrating with GitHub.

As GitHub is most likely the top use case, I think it might be beneficial to have an entire row dedicated to Source Control and the VCS' as tiles?